storage/badger/dkg_state_test.go

```diff
@@ -344,9 +515,16 @@ func TestDKGState_RandomBeaconKeyCommittedState(t *testing.T) {
 		store, err := NewRecoverableRandomBeaconStateMachine(metrics, db)
 		require.NoError(t, err)
 
+		var evidence *flow.EpochCommit
+		var pkey crypto.PrivateKey
```

Comment:
```suggestion
		var privKey crypto.PrivateKey
```
Nitpick: avoiding just the "p" prefix (because public also starts with "p" 😄)